dev-libs/libedit/libedit-20170329.3.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit eutils flag-o-matic toolchain-funcs versionator multilib-minimal
+
+MY_PV=$(get_major_version)-$(get_after_major_version)
```

Comment:
MY_PV=${PV/./-}

would be equivalent, wouldn't it? If so, you can avoid using versionator (i.e. save code).